drivers/flash/nrf_qspi_nor.c

```diff
@@ -1115,16 +1110,6 @@ static int qspi_nor_init(const struct device *dev)
 		(void)nrfx_qspi_deactivate();
 	}
 
-#ifdef CONFIG_PM_DEVICE_RUNTIME
-	int rc2 = pm_device_runtime_enable(dev);
-
-	if (rc2 < 0) {
-		LOG_ERR("Failed to enable runtime power management: %d", rc2);
-	} else {
-		LOG_DBG("Runtime power management enabled");
-	}
-#endif
```

Comment:
Shouldn't this be replaced with `pm_device_runtime_auto_enable()` so the `zephyr,pm-device-runtime-auto` property is applied?

Reply:
No, the whole point is that drivers don't need to know about it:
https://github.com/zephyrproject-rtos/zephyr/blob/fd0e46506fabd4c0c91ecb336a39cb6a6aa2b270/kernel/init.c#L330-L331